nevergrad/benchmark/plotting.py

```diff
@@ -313,6 +313,10 @@ def create_plots(
             print("\n# new case #", fixed, case)
             casedf = df.select(**dict(zip(fixed, case)))
             data_df = FightPlotter.winrates_from_selection(casedf, fight_descriptors, num_rows=num_rows)
+            # We also want a bigger dataframe with more columns for writing a text overview.
+            data_df_big = FightPlotter.winrates_from_selection(
+                casedf, fight_descriptors, num_rows=2, num_cols=100
+            )
```

Comment:
this should not be called twice because it's slow, can we get the big one, and then cut the smaller one out of it?

Reply:
Ok: only one now. 
We print only the 30 best then, which is ok for now.